_format/1.1/index.md

```diff
@@ -1803,6 +1803,9 @@ An error object **MAY** have the following members:
 * `links`: a [links object][links] containing the following members:
   * `about`: a [link][links] that leads to further details about this
     particular occurrence of the problem.
+  * `type`: a [link][links] whose URI that identifies the type of error that 
```

Comment:
> whose URI that identifies

Use of `whose` + `that` is a little awkward. Could we just strike "whose URI" entirely?